schema/verify/primer_search.sql

```diff
@@ -0,0 +1,9 @@
+-- Verify primer_position_search
+
+BEGIN;
+
+SELECT
+    1/pg_catalog.has_table_privilege('viroverse_w', 'viroserve.primer_search', 'INSERT')::int +
```

Comment:
update this to `SELECT` if modifying the grant to `viroverse_w` in the deployment script. Also add a similar check for `viroverse_w` `EXECUTE` privs on the refresh function.